docs/_includes/header.html

```diff
@@ -59,11 +59,12 @@
       name="twitter:image"
       content="https://getnighthawk.dev/images/logos/nighthawk/icon/nighthawk-logo.svg"
     />
+    <!--
     <meta
       name="image"
       property="og:image"
       content="https://getnighthawk.dev/images/logos/nighthawk/icon/nighthawk-logo.svg"
-    />
+    /> -->
```

Comment:
@Vidyaa123  when I am using this deployed link by Netlify on https://www.opengraph.xyz/ it shows rich previews but when I try the same link on actual platforms like Twitter or WhatsApp it is unable to show image preview, so can you please try to insert and test this code on you header.html file once so that we can inspect the issue 

```
<!-- HTML Meta Tags -->
<title>Nighthawk</title>
<meta name="description" content="The Nighthawk distribution for service mesh performance characterization. Integrating Meshery and Service Mesh Performance.">

<!-- Facebook Meta Tags -->
<meta property="og:url" content="https://deploy-preview-236--getnighthawk.netlify.app/">
<meta property="og:type" content="website">
<meta property="og:title" content="Nighthawk">
<meta property="og:description" content="The Nighthawk distribution for service mesh performance characterization. Integrating Meshery and Service Mesh Performance.">
<meta property="og:image" content="https://getnighthawk.dev/assets/images/logos/nighthawk/icon/nighthawk-logo.png">

<!-- Twitter Meta Tags -->
<meta name="twitter:card" content="summary_large_image">
<meta property="twitter:domain" content="deploy-preview-236--getnighthawk.netlify.app">
<meta property="twitter:url" content="https://deploy-preview-236--getnighthawk.netlify.app/">
<meta name="twitter:title" content="Nighthawk">
<meta name="twitter:description" content="The Nighthawk distribution for service mesh performance characterization. Integrating Meshery and Service Mesh Performance.">
<meta name="twitter:image" content="https://getnighthawk.dev/assets/images/logos/nighthawk/icon/nighthawk-logo.png">

```

Reply:
@Abhijay007, I will check and revert.

Reply:
@Abhijay007, The above code works in twitter validator now. Please review.

Reply:
@vinayaksh42,  I can see the recommended image size is 1200 x 630.  I set it up previously and it didn't work properly. Will check again.

Reply:
@Abhijay007 [opengraph](https://www.opengraph.xyz) is one of the publicly available debugging tools which can be used to generate rich previews, the open graph [protocol](https://ogp.me) or simply `og:image` tag is a global structure and doesn't get updated with new Meta's new releases which have their own set of rules governing around the same. Moreover, Twitter doesn't use og internally, hence the previews (may) differ by a few pixel shifts.

@Vidyaa123 Try Meta's official debugging tool to get the best preview which can be accessed from [here](https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fdeploy-preview-236--getnighthawk.netlify.app%2F). Note that I've already set the endpoint to the Netlify's [latest](https://github.com/layer5io/getnighthawk/pull/236#issuecomment-1008680158) build preview.

I tried on both mobile & Mac, checks out that the image doesn't fit properly on Facebook. The same is also applicable for Twitter rp too. Attached the below screenshot for reference 👇 

![image](https://user-images.githubusercontent.com/48355572/149142284-e3e6ae05-d02f-4431-ad79-7ab960fddebb.png)

Hence, I'd recommend fitting the image by either editing a copy of source-image of Nighthawk's logo by taking @leecalcote's permission. Lmk when you're done & feel free to ask questions if get stuck (:

Reply:
& yes, discord seems all fine!

![image](https://user-images.githubusercontent.com/48355572/149145998-38039f80-31f0-4d77-bb9c-23a6f84bb865.png)

Reply:
Thanks, @Neilblaze. @leecalcote Can you please permit me to edit a copy of nighthawk's logo? Thanks

Reply:
> @Abhijay007 [opengraph](https://www.opengraph.xyz) is one of the publicly available debugging tools which can be used to generate rich previews, the open graph [protocol](https://ogp.me) or simply `og:image` tag is a global structure and doesn't get updated with new Meta's new releases which have their own set of rules governing around the same. Moreover, Twitter doesn't use og internally, hence the previews (may) differ by a few pixel shifts.

Thanks, @Neilblaze for this info, I will keep that in mind :)